SU-HSDO/suhumsci
# More global and France-Stanford Tweaks

# READY FOR REVIEW

# Summary
- Tweaks to three column with image layout
- Masonry item tweaks and France-Stanford specific Vertical Card on project page tweaks
- Increase spacing between sub-navigation and exposed filter(s)
- Increase spacing between paragraph items

# Needed By (Date)
- End of sprint ()

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that above changes are evident (I can provide link examples as needed)

# Affected Projects or Products
- France-Stanford
- suhumsci (theme) and sites using it

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-419
- https://stanfordits.atlassian.net/browse/HSD8-420
- https://stanfordits.atlassian.net/browse/HSD8-421
- https://stanfordits.atlassian.net/browse/HSD8-422
- Build on top of PR at https://github.com/SU-HSDO/suhumsci/pull/112

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
